lambda/translcore.ml

```diff
@@ -901,6 +902,7 @@ and transl_tupled_cases ~scopes patl_expr_list =
   List.map (fun (patl, guard, expr) -> (patl, transl_guard ~scopes guard expr))
     patl_expr_list
 
+(* FIXME: Does this respect Rc_nontail? *)
```

Comment:
I think it's safe as it is, but `[@nontail]` will prevent an application from being merged with following application.